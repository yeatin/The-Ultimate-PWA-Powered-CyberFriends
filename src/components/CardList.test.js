import { shallow } from "enzyme";
import CardList from "./CardList";
import React from "react";

it("expect to render CardList component", () => {
    const mockRobots = [{
        id: 1,
        name: 'John Cena',
        email: 'BingChilling@gmail.com'
    }]
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
})