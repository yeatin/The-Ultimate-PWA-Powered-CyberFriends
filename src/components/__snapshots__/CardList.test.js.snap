// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <Card
    email="BingChilling@gmail.com"
    id={1}
    key="1"
    name="John Cena"
  />
</div>
`;
